learn/hoon/hoon-tutorial/hoon-syntax.md

```diff
@@ -127,7 +123,6 @@ Let's start with the base case: a single limb.  A limb expression is a trivial w
 - `b`
 - `add`
 - `mul`
-- `kebab-case-123`
```

Comment:
Is the permissibility of dashes in identifiers just getting dropped from the syntax doc?

Reply:
This was part of an economization/readability effort. The doc got longer and more unwieldy